pkg/export/export.go

```diff
@@ -70,6 +70,10 @@ var (
 		Name: "gcm_export_samples_sent_total",
 		Help: "Number of exported samples sent to GCM.",
 	})
+	samplesSentErrors = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Let's not forget to register that metric

Reply:
Good catch - done!